packages/common/src/schemas/documents/default-context-schema.json

```diff
@@ -167,7 +167,15 @@
                   "additionalProperties": false
                 }
               },
-              "required": ["provider", "uri", "type"],
+              "required": ["provider", "type"],
```

Comment:
This file is deprecated, deleted it from dev. use the typebox version in the same folder

Reply:
Typebox version is used during `seed`